data/mods/Dark-Skies-Above/blacklists/trait_blacklist.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "//": "no need for robust genetics atm",
+    "type": "ITEM_BLACKLIST",
+    "traits": [ "ROBUST" ]
```

Comment:
![image](https://user-images.githubusercontent.com/30208401/78186455-c75e6980-743a-11ea-9b5e-1b25560c4cd1.png)

Reply:
This was supposed to be a TRAIT_BLACKLIST, fixed.